### PR TITLE
Lps 129061 Fix empty list when getting permissions and add actions

### PR DESCRIPTION
**Pull request by javierdearcos**
As we talked about, I fix the empty list when getting the permissions of a resource that hasn't have the permissions created, following the same approach we are using in the GUI and you point me to.

I've also added the actions to the Permissions Page as it was easy so we can avoid regenerating again the code in every branch.

Pending to test in all branches before forwarding the PR

### CODE REVIEW

**Review by nhpatt — APPROVED** [review]

**Review by nhpatt — COMMENTED** [review]
Remember to add the label

File: modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl [hunk 1]
```diff
@@ -188,7 +189,12 @@ public abstract class Base${schemaName}ResourceImpl
 
 					PermissionUtil.checkPermission(ActionKeys.PERMISSIONS, groupLocalService, resourceName, resourceId, getPermissionCheckerGroupId(${schemaVarName}Id));
 
-					return toPermissionPage(resourceId, resourceName, roleNames);
+					return toPermissionPage(
+						<@permissionActions
```
[the remark this answers is not in the document]
Comment (javierdearcos) [reply modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl:193]: 😄  Yes! That can be a great step toward readability in our templates

**Comment by javierdearcos** [comment]
> Remember to add the label

Sure! I was not sure about it because I didn't know if it was put automatically or if I need special permissions. I will put it from now on

**Comment by javierdearcos** [comment]
ci:test:sf

**Comment by javierdearcos** [comment]
Testing the other branches before forwarding it. I'll execute the tests meanwhile to try to forward it immediatelly

**Comment by javierdearcos** [comment]
ci:test:sf

**Comment by javierdearcos** [comment]
ci:test:relevant

**Comment by javierdearcos** [comment]
ci:stop

**Comment by javierdearcos** [comment]
I've just checked and the generation is OK in the other branches. Just 7.3.x have changes BTW

Just to be sure @nhpatt. As this PR introduces a new method in PermissionUtil that is used by the autogeneration it would be a breaking change in the RESTBuilder-Vulcan dependency. In this case, I think it doesn't matter because it affects only the permission endpoint autogeneration, that is something internal that is very difficult that an external client uses. But if you think this could be problematic, I can think of a kind of Feature Flag implementation in the templates to avoid this (and that can be useful in the future)

**Comment by nhpatt** [comment]
I don't think no-one except us uses it, it's completely buried.

**Comment by nhpatt** [comment]
ci:forward

**Comment by nhpatt** [comment]
ci:forward

**Comment by nhpatt** [comment]
ci:forward

**Comment by javierdearcos** [comment]
Seems not related... Running tests again

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:reevaluate:1117401_8942

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:forward

**Comment by nhpatt** [comment]
The commit message has the LPS duplicated: https://github.com/liferay-frontend/liferay-portal/pull/1074/commits/691e20ecc7c25ac05e827505d1ba6995c907c83c

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:stop

**Comment by javierdearcos** [comment]
Sent here: https://github.com/liferay-frontend/liferay-portal/pull/1101

**Pull request closed without merging**